web/src/components/dashboard/SidebarCardsContainer.tsx

```diff
@@ -28,10 +28,10 @@ export const SidebarCardsContainer = (props: Props): ReactElement => {
 
   const visibleSidebarCards = getVisibleSideBarCards(business, props.sidebarDisplayContent);
   const visibleSortedFundings = getVisibleFundings(
-    sortFundings(filterFundings({ fundings: props.fundings, business: business }), userData),
+    sortFundingsForUser(filterFundings({ fundings: props.fundings, business: business }), userData),
```

Comment:
Addressing this comment from InvestNewark related PR: https://github.com/newjersey/navigator.business.nj.gov/pull/8339/files#r1700748265